pages/common/odps-resource.md

```diff
@@ -0,0 +1,26 @@
+# odps resource
+
+> Manage resources in odps.
+
+- Show resources in the current project:
+
+`list resources;`
+
+- Add local file as odps resource (overwrite if it already exists):
+
+`add file {{file}} as {{alias}} -f;`
+`add archive {{file}} as {{alias}} -f;`
+`add jar {{file.jar}} -f;`
+`add py {{file.py}} -f;`
```

Comment:
We typically don't allow multiple commands per example (see #687 and #699). You could either separe this as multiple examples, use a single example of these, or build the command using generic tokens rather than specific parameters.